content/integrations/azure.md

```diff
@@ -73,3 +121,4 @@ Please make sure you have enabled diagnostics on your VMs. Diagnostics allows VM
    [2]: https://portal.azure.com
    [3]: mailto:support@datadoghq.com
    [4]: https://management.azure.com
+   [5]: https://manage.windowsazure.com
```

Comment:
@DanielLanger let us know if you need help with this one, but it should just be adding that macro matt provided at the bottom.